shared/modules/testcheck_module.py

```diff
@@ -23,7 +23,7 @@
        <generator>
         <oval:product_name>testcheck.py</oval:product_name>
         <oval:product_version>0.0.1</oval:product_version>
-        <oval:schema_version>5.10</oval:schema_version>
+        <oval:schema_version>5.11</oval:schema_version>
```

Comment:
I realized that this must be done differently. This would affect also the RHEL content. It should be re-factored to influence only the Fedora content.